### PR TITLE
Bumped ids and added designer.

### PR DESCRIPTION
**Pull request by bsthomsen**

### CODE REVIEW

File: employees.js [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
-    { id: 12, name: 'Rosie	Porter', experience: 246400, skill: 50, type: EmployeeTypes.ProjectManager, image: 'images/avatars/female/3.png', salary: 5000, skills:[], avatar: 'female1', progress: 0, happiness: 50 },
+    { id: 4001, name: 'Marlene Green', experience: 162500, skill: 40, type: EmployeeTypes.ProjectManager, image: 'images/avatars/female/2.png', salary: 3700, skills:[], avatar: 'female1', progress: 0, happiness: 50 },
+    { id: 4002, name: 'Howard Hampton', experience: 70400, skill: 30, type: EmployeeTypes.ProjectManager, image: 'images/avatars/male/3.png', salary: 3000, skills:[], avatar: 'male1', progress: 0, happiness: 50 },
+    { id: 4003, name: 'Rosie Porter', experience: 246400, skill: 50, type: EmployeeTypes.ProjectManager, image: 'images/avatars/female/3.png', salary: 5000, skills:[], avatar: 'female1', progress: 0, happiness: 50 },
+
+    // Designers
+    { id: 4001, name: 'Farisha', experience: 70400, skill: 30, type: EmployeeTypes.Designer, image: 'images/avatars/female/2.png', salary: 1800, skills:[4001], avatar: 'female1', progress: 0, happiness: 50 },
```
Comment (tamtamchik) [inline employees.js]: duplicate! @bsthomsen with `'Marlene Green'`

Comment (bsthomsen) [inline employees.js]: @tamtamchik yea trying to change the general id structure so that it's easier to add a new employee.

Comment (tamtamchik) [inline employees.js]: I see, and it's good, just if `4001` duplicated for  `Farisha` and `Marlene Green` :)

Comment (bsthomsen) [inline employees.js]: Ohh! i will change that :)

**Comment by tamtamchik** [comment]
:+1:

**Comment by bsthomsen** [comment]
@tamtamchik check now :)

**Pull request merged**